orttraining/orttraining/python/checkpointing_utils.py

```diff
@@ -0,0 +1,239 @@
+import os
+from collections import OrderedDict
+import torch
+
+def list_checkpoint_files(checkpoint_dir, checkpoint_prefix, extension='.tar'):
+    ckpt_file_names = [f for f in os.listdir(checkpoint_dir) if f.startswith(checkpoint_prefix)]
+    ckpt_file_names = [f for f in ckpt_file_names if f.endswith(extension)]
+    ckpt_file_names = [os.path.join(checkpoint_dir, f) for f in ckpt_file_names]
+    
+    assert len(ckpt_file_names) > 0, "No checkpoint files found with prefix \"{}\" in directory {}.".format(checkpoint_prefix, checkpoint_dir)
+    return ckpt_file_names
+
+class Combine_Zero_Checkpoint():
```

Comment:
Python classes usually dont have underscore in their names and the best practice is to inherit from `object` when no for base class (the one that doesnt have any parent class). How about `CombineZeroCheckpoint(object)`.